packages/3id-did-resolver/src/index.ts

```diff
@@ -190,6 +190,7 @@ const resolve = async (
     query.push({
       streamId,
       atTime: verNfo.timestamp,
+      opts: { atTime: verNfo.timestamp },
```

Comment:
Please, create an issue to remove the deprecated `atTime` after sufficient number of nodes got updated.